src/systems/filecoin_markets/deal/_index.md

```diff
@@ -65,3 +65,34 @@ On every PoSt Submission, the following steps happen.
 # Deal Flow
 
 {{< diagram src="diagrams/deal-flow.mmd.svg" title="Deal Flow Sequence Diagram" >}}
+
+# Deal States
+
+All on-chain economic activities in Filecoin start with the deal. This section aims to explain different states of a deal and their relationship with other concepts in the protocol such as Power, Payment, and Collaterals.
+
+A deal has the following states:
+
+- `Unpublished`: the deal has yet to be posted on chain.
+- `Published`: the deal has been published and accepted by the chain but is not yet active as the sector containing the deal has not been proven.
```

Comment:
`staged`? just a proposal

Reply:
lets stick with Unpublished, it also includes deals that are not yet created and hence not yet staged.